src/dialects/abstract/query-interface.d.ts

```diff
@@ -280,7 +281,7 @@ export interface RemoveColumnOptions extends RemoveColumnQueryOptions, QueryRawO
 * This interface is available through sequelize.queryInterface. It should not be commonly used, but it's
 * referenced anyway, so it can be used.
 */
-export class QueryInterface {
+export class QueryInterface extends AbstractQueryInterfaceTypeScript {
```

Comment:
don't forget to remove create schema from this file, since it's now in the parent class